ludwig/schema/hyperopt/hyperopt.py

```diff
@@ -0,0 +1,97 @@
+from abc import ABC
+from typing import Any
+
+from marshmallow_dataclass import dataclass
+
+from ludwig.constants import LOSS
+from ludwig.features.feature_registries import output_type_registry
+from ludwig.schema import utils as schema_utils
+from ludwig.schema.hyperopt.executor import ExecutorConfig, ExecutorDataclassField
+from ludwig.schema.hyperopt.search_algorithm import SearchAlgorithmConfig, SearchAlgorithmDataclassField
+
+# hyperopt:
```

Comment:
Do we need this commented out code here still?

Reply:
nope!